components/automate-cli/cmd/chef-automate/ha_node_operations_utils.go

```diff
@@ -321,6 +327,60 @@ func (nu *NodeUtilsImpl) executeShellCommand(command, path string) error {
 	}, path)
 }
 
+// Stop services on node
+func (nu *NodeUtilsImpl) stopServicesOnNode(ip, nodeType string, infra *AutomateHAInfraDetails, sshUtil SSHUtil) error {
```

Comment:
modify it to work for AWS deployment also.

Reply:
done